packages/plugin-commands-init/src/init.ts

```diff
@@ -0,0 +1,51 @@
+import fs from 'fs'
+import path from 'path'
+import { docsUrl } from '@pnpm/cli-utils'
+import { UniversalOptions } from '@pnpm/config'
+import PnpmError from '@pnpm/error'
+import writeProjectManifest from '@pnpm/write-project-manifest'
+import renderHelp from 'render-help'
+
+export const rcOptionsTypes = cliOptionsTypes
+
+export function cliOptionsTypes () {
+  return {}
+}
+
+export const commandNames = ['init', 'create', 'innit']
+
+export function help () {
+  return renderHelp({
+    description: 'Create a package.json file',
+    descriptionLists: [],
+    url: docsUrl('init'),
+    usages: ['pnpm init'],
+  })
+}
+
+export async function handler (
+  opts: Pick<UniversalOptions, 'dir'>
+) {
+  const manifestPath = path.join(opts.dir, 'package.json')
+  if (fs.existsSync(manifestPath)) {
+    throw new PnpmError('PACKAGE_JSON_EXISTS', 'package.json already exists')
+  }
+  const manifest = {
+    name: path.basename(opts.dir),
+    version: '0.0.0',
```

Comment:
Perhaps it would be better to use `1.0.0` or `0.0.1`

The advantage is that some developers can use the `pulish` command directly without reporting an error.